.travis.yml

```diff
@@ -32,6 +32,10 @@ env:
       - "TEST_PATTERN=tests/test_harness.py::TestDestroyOnSuccess::test_default"
       - "TEST_PATTERN=tests/test_harness.py::TestDestroyOnSuccess::test_false"
       - "TEST_PATTERN=tests/backends/test_docker.py::TestCustomMasterMounts::test_custom_master_mounts"
+
+      - "TEST_PATTERN=tests/backends/test_docker.py::TestBadParameters::test_no_build_artifact"
```

Comment:
Each builder takes about 5 minutes and we have a limited number of builders available. I'd copy the format for the other `TestBadParameters` below and put these all on one builder, with a note.